app/src/main/java/org/dslul/openboard/inputmethod/latin/settings/UserDictionarySettings.java

```diff
@@ -139,30 +138,24 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         listView.setFastScrollEnabled(true);
         listView.setEmptyView(emptyView);
 
-        setHasOptionsMenu(true);
-    }
-
-    @Override
-    public void onStart() {
-        super.onStart();
-        // Show the language as a subtitle of the action bar
         final ActionBar actionBar = ((AppCompatActivity) requireActivity()).getSupportActionBar();
         if (actionBar == null) return;
-        actionBar.setSubtitle(UserDictionarySettingsUtils.getLocaleDisplayName(getActivity(), mLocale));
+        // Special treatment for the known languages with _zz and _ZZ types
+        if (mLocale.endsWith("_zz") || mLocale.endsWith("_ZZ")) {
+            final int resId = requireContext().getResources().getIdentifier("subtype_"+mLocale, "string", requireContext().getPackageName());
+            actionBar.setSubtitle(requireContext().getString(resId));
+        } else {
+            actionBar.setSubtitle(getLocaleDisplayName(getActivity(), mLocale));
```

Comment:
same thing with `getLocaleDisplayNameInSystemLocale` seems to apply here

Reply:
Done with commit f7f791c
As we don't have access to _Locale_ I updated `UserDictionarySettings.getLocaleDisplayName()` instead and applied it where necessary.